python/packages/ai/teams/ai/models/prompt_response.py

```diff
@@ -11,7 +11,9 @@
 from ..prompts.message import Message
 
 ContentT = TypeVar("ContentT")
-PromptResponseStatus = Literal["success", "error", "rate_limited", "invalid_response", "too_long"]
+PromptResponseStatus = Literal[
+    "success", "error", "rate_limited", "invalid_response", "too_long", "tools_error"
```

Comment:
we should not use `tools` terminology` when possible since its a concept of openai and not necessarily all language models. Is there a reason we can't reuse `error` ?